plugins/hls-tactics-plugin/test/golden/DestructTyFam.hs

```diff
@@ -0,0 +1,8 @@
+{-# LANGUAGE TypeFamilies #-}
+
+type family Yo where
+  Yo = Bool
+
+test :: Yo -> Int
+test b = _
+
```

Comment:
Does this work with open type family too?

Reply:
Added some tests

Reply:
Awesome, thanks!